client/state/selectors/get-site-comments.js

```diff
@@ -20,16 +20,17 @@ function filterCommentsByStatus( comments, status ) {
  * @param {Object} state Redux state
  * @param {Number} siteId Site for whose comments to find
  * @param {String} [status=unapproved] Status to filter comments
+ * @param {String} [order=asc] Order in which to sort filtered comments
  * @returns {Array<Object>} Available comments for site, filtered by status
  */
 export const getSiteComments = createSelector(
-	( state, siteId, status = 'unapproved' ) => {
+	( state, siteId, status = 'unapproved', order = 'asc' ) => {
```

Comment:
Once we get past 3, or 4 args, it's usually a smell to use an options object. There's more context for the reader, and we're less likely to have an error where args are off by one.